views/user_sessions/new.haml

```diff
@@ -0,0 +1,9 @@
+.login-form
+  %form.form-signin{action: '/user_sessions/create', method: 'POST'}
+    =Rack::Csrf.tag(env)
+    %h2.form-signin-heading Please sign in
+    %label.sr-only{:for => "email"} Email address
+    %input.form-control{:autofocus => "", :name => "email", :placeholder => "Email address", :required => "", :type => "email"}/
+    %label.sr-only{:for => "password"} Password
+    %input.form-control{:name => "password", :placeholder => "Password", :required => "", :type => "password"}/
```

Comment:
Line is too long. [111/80]<br>Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace